### PR TITLE
Sleeping Carp attacks now have attack animations

### LINKED ISSUE #14530: Sleeping carp attacks have no animation.
There's nothing to indicate you're hitting people once you've used the scroll. I'm not even sure if the special attacks them self work.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #14530

### CODE REVIEW

**Comment by Xhuis** [comment]
When I first made this, attack animations didn't exist. This is a very welcome change.
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by xxalpha** [comment]
Can you make them spin and rotate like eswords do?

**Comment by RemieRichards** [comment]
@Xhuis that's not true.
You added sleeping carp on the 27th of Jun 2015.
Attack animations got updated on the 3rd of November 2014.

**Comment by Xhuis** [comment]
You might be right, @RemieRichards, but I still recall not knowing about attack animations when I made Sleeping Carp. Oh well, it won't matter anyway if this gets merged.

**Comment by RemieRichards** [comment]
Yeah, not /knowing about them/ is a whole nother bowl of fruit, so fair's fair.

File: code/datums/martial.dm [hunk 1]
```diff
@@ -319,6 +319,7 @@
 
 /datum/martial_art/the_sleeping_carp/proc/wristWrench(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	if(!D.stat && !D.stunned && !D.weakened)
+		A.do_attack_animation(D)
```
Comment (KorPhaeron) [inline code/datums/martial.dm]: Is there not a way to just have this happen anytime a combo is pulled off instead of pasting it in each combo?

**Pull request merged**